### PR TITLE
Enhancement: Leverage GitHub issue and PR template capabilities

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] leverages GitHub issue and PR template capabilities

💁 For reference, see https://github.com/blog/2111-issue-and-pull-request-templates.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ composer dump-autoload
 
 ## Contributing
 
-For us to accept contributions you will have to first have signed the [Contributor License Agreement](https://code.facebook.com/cla). Please see [CONTRIBUTING](https://github.com/facebook/facebook-instant-articles-sdk-php/blob/master/CONTRIBUTING.md) for details.
+For us to accept contributions you will have to first have signed the [Contributor License Agreement](https://code.facebook.com/cla). Please see [CONTRIBUTING](https://github.com/facebook/facebook-instant-articles-sdk-php/blob/master/.github/CONTRIBUTING.md) for details.
```
Comment (localheinz) [inline README.md]: Unrelated, but wondering - why are the links to files contained within this repository made absolute?

**Comment by localheinz** [comment]
Build is failing, you should probably build only when `.travis.yml` is present (or add it):

screen shot 2016-04-19 at 19 34 57

See https://travis-ci.org/facebook/facebook-instant-articles-sdk-php/settings.

**Comment by everton-rosario** [comment]
@localheinz this wont work.
We want this MD to be a navigate link, so if you paste this link in your browser it works:
- https://github.com/facebook/facebook-instant-articles-sdk-php/blob/master/CONTRIBUTING.md

Thus, this link with ".github" you've added, wont work for this browser navigation.
- https://github.com/facebook/facebook-instant-articles-sdk-php/blob/master/.github/CONTRIBUTING.md

We are working on a better documentation for this SDK, then this web links between MD wont be necessary. But until it works, lets keep it as it is.

Thanks!

**Comment by localheinz** [comment]
@everton-rosario 

Well, we can still make use of GitHub issue and pull requests templates, there's no need for `CONTRIBUTING.md` to live in `.github`.

**Comment by localheinz** [comment]
@everton-rosario 

I've amended the commit, care to take a look again?

**Pull request closed without merging**